src/policies/online_gae.jl

```diff
@@ -96,7 +96,7 @@ We assume the general form of mapping states (s) to actions (a):
     - the sufficient statistics of distribution D
     - some concatenated combination of μ/U/σ for multivariate normals
 """
-type OnlineGAE{T      <: Number,
+mutable struct OnlineGAE{T      <: Number,
```

Comment:
align arguments

Reply:
bad bot

Reply:
You need to make comments with the "changes requested" review feature for the bot to act.

Reply:
:-o